lua/telescope/actions/init.lua

```diff
@@ -272,6 +272,24 @@ function actions.close_pum(_)
   end
 end
 
+--- Copy current entry command to a specific register or unnamed plus as default
+--- Don't let user use  blackhole register (_), fallback to default
+---@param prompt_bufnr number: The prompt bufnr
+---@param opts table: options to pass to the action
+---@field reg: the register which the content will be sent
+function actions.copy_command_to_reg(prompt_bufnr, opts)
+  opts = opts or {} -- ensure opts is a table if nil
+  opts.reg = utils.get_default(opts.reg, "+")
+  if opts.reg == _ then
```

Comment:
Should be "_" I think in any case.

Reply:
Hum, not sure what I am doing wrong, but since .I started using efm to format lua code things like that started to happen (e.g removing "").